CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,217 @@
+# Quansight Repository Code of Conduct
+
+1. Introduction
+    a. Expected Behavior
+    b. Unacceptable Behavior
+2. Incident Reporting Guidelines
+    a. How to Submit an Incident Report
+    b. Conflicts of Interest
+    c. What to Include in a Report
+    d. Procedures after Filing a Report
+3. License
+
+-------
+
+## 1. Introduction
+
+The aim of this Code of Conduct is to empower all participants in Quansight-managed public repositories to facilitate healthy and constructive community behavior. Those participants may be users, maintainers, or contributors to open-source projects managed by Quansight LLC or Quansight Labs. This Code of Conduct outlines expected behaviors as well as steps for reporting incidents of unacceptable behavior.
+
+This Code of Conduct applies to all repositories and communities for Quansight-managed open-source projects regardless of whether or not the repository explicitly includes this document. The Code also applies in public spaces when an individual represents a project or its community. Examples include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or in-person event. What constitutes representing a project may be further defined and clarified by project maintainers.
+
+Quansight is committed to providing a welcoming and inspiring community for all. Anyone claiming affiliation with Quansight public repositories is required to conform to the Code of Conduct. This Code of Conduct applies to all spaces and public repositories of Quansight on all platforms (including Github). Possible consequences for breaching this Code of Conduct are listed below.
+
+For any queries regarding this Code of Conduct, please contact any member of the Code of Conduct committee. The Code of Conduct committee (for 2021) is:
+
++ Name 1
++ Name 2
++ Name 3
+
+The COC committee consists of (at least) three members of Quansight staff. The term of the membership is one calendar year with the option for renewal if desired (the COC committee may stagger departures to preserve committee memory)
+
+You may alternatively choose to contact one of the Quansight Labs Co-Directors with questions regarding this COC:
+
++ Ralf Gommers
++ Tania Allard
+
+### a. Expected Behavior
+
++ Be empathetic, welcoming, friendly, patient, considerate, and mindful.
+    + Everyone deserves to be treated with kindness and empathy.
++ Consider the consequences of any decision you make on users & colleagues.
```

Comment:
This doesn't look like it belongs here, sounds like that's about technical things. Also, users aren't directly involved in CoC-related topics (users consume passively, CoC is about interactions between active participants).

Reply:
> + ~Consider the consequences of any decision you make on users & colleagues.~
>      + ~Your work will be used by others and you will depend on the work of others~

I assume you are referring to the preceding two lines (which have now been struck).

Reply:
yes indeed